src/lecture_feedback/room.py

```diff
@@ -9,9 +9,6 @@ def __init__(self, room_id: str) -> None:
         self._room_id = room_id
         self._sessions: ThreadSafeDict[UserStatus] = ThreadSafeDict()
 
-    def get_session_status(self, session_id: str) -> UserStatus:
```

Comment:
Unused function, grep didn't return any other instance of string `get_session_status` in the codebase